spec/octokit/client/repositories_spec.rb

```diff
@@ -128,11 +128,11 @@
 
     describe '.create_repository_from_template', :vcr do
       before do
-        @client.edit_repository(@repo.full_name, is_template: true)
+        @client.edit_repository(@repo.full_name, is_template: true, accept: Octokit::Preview::PREVIEW_TYPES.fetch(:template_repositories))
       end
 
       it 'generates a repository from the template' do
-        @client.create_repository_from_template(@repo.id, 'Cloned repo')
+        @client.create_repository_from_template(@repo.id, 'Cloned repo', accept: Octokit::Preview::PREVIEW_TYPES.fetch(:template_repositories))
```

Comment:
Note: API previews are no longer supported/used in the [api.github.com](https://github.com/github/rest-api-description/tree/main/descriptions/api.github.com) and [ghes-3.3](https://github.com/github/rest-api-description/tree/main/descriptions/ghes-3.3) and above for the REST API.  

In fact, all of them have been deprecated, which effectively makes this change a no-op.  

I have created an [issue](https://github.com/octokit/octokit.rb/issues/1447) for this work. This is not a show stopper for this PR, given we'll get to remove all of this when we do the cleanup.